config.py

```diff
@@ -20,7 +20,13 @@ class Config(object):
         "DATABASE_URL"
     ) or "sqlite:///" + os.path.join(basedir, "app.db")
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-
+    # psotgresql crednetials. If you want to use postgres db, uncomment the following lines and comment out  above  SQLALCHEMY_DATABASE_URI corresponding to sqlite
+    #hostname = os.getenv("DB_HOSTNAME")
+    #user = os.getenv("USERNAME_DB")
+    #database = os.getenv("DATABASE")
+    #password = os.getenv("PASSWORD")
+    #port = os.getenv("PORT")
+    #  SQLALCHEMY_DATABASE_URI = f"postgresql://{user}:{password}@{hostname}:{port}/{database}"
```

Comment:
Idealy we'd make this a bit more dynamic, maybe something like Miguel Grinberg does in his Flask Mega Tutorial:

```
class Config(object):
    # ...
    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL', '').replace(
        'postgres://', 'postgresql://') or \
        'sqlite:///' + os.path.join(basedir, 'app.db')
    # ...
```

See here for the full explanation: https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xviii-deployment-on-heroku

Reply:
Personally I would also go with option 1 since I expect this template to be used by people deploying on Heroku and Render. So incorporate the example I shared to maximize compatibility with deployment platforms and have end-user set the DATABASE_URL env variable.